docs/content/TopNQuery.md

```diff
@@ -211,4 +211,7 @@ Users who can tolerate *approximate rank* topN over a dimension with greater tha
     "queryType": "topN",
     "threshold": 2
 }
-```
\ No newline at end of file
+```
+
+### DimExtractionFn
```

Comment:
Added it anyways, turned out not to be too bad if it is handled at the premerge / postmerge decorators which are only called by the broker.